### PR TITLE
Update Apache commons-codec to latest version (1.14)

### PR DESCRIPTION
**Pull request by ayoho**

### CODE REVIEW

**Comment by ayoho** [comment]
#build

File: dev/cnf/oss_dependencies.maven [hunk 1]
```diff
@@ -90,6 +90,7 @@ commons-beanutils:commons-beanutils:1.9.4
 commons-cli:commons-cli:1.4
 commons-codec:commons-codec:1.10
 commons-codec:commons-codec:1.11
+commons-codec:commons-codec:1.14
```
Comment (tevans78) [inline dev/cnf/oss_dependencies.maven:93]: Why do we need to keep the old versions as well?

Comment (ayoho) [reply dev/cnf/oss_dependencies.maven:93]: I don't know everyone who might be using some of the old versions, but I know some FATs use some of the older libraries. I don't want to dive into making sure everyone's happy and working with the latest version just to get this upgrade delivered.

**Comment by ayoho** [comment]
#build

**Comment by ayoho** [comment]
#build

**Pull request merged**